step-templates/upgrade-octopus-server.json

```diff
@@ -22,13 +25,23 @@
         "Octopus.ControlType": "SingleLineText"
       },
       "Links": {}
+    },
+    {
+      "Id": "18ede5ad-24a8-4796-a6e2-1b49a4cd7df2",
+      "Name": "InstallPath",
+      "Label": "InstallPath",
+      "HelpText": "The installation path to Octopus Deploy Server. Not the instance directory",
+      "DefaultValue": "C:\\Program Files\\Octopus Deploy\\Octopus",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
     }
   ],
-  "LastModifiedBy": "droyad",
```

Comment:
The build is failing due to this field missing, please add it back in and you can update it with your :octocat: handle and it'll be properly linked on the Library site

Reply:
done :) my bad!